src/schema.ts

```diff
@@ -40,79 +49,10 @@ export type DrumType = "bassdrum" | "hihat" | "snare";
  */
 export type Channel = DrumType | "vibraphone" | "bass";
 
-/**
- * Represents a program on the MMX's programming wheel. Used to store the programmed part of a performance.
- */
-export interface Program {
-	/**
-	 * Information about this program. It can be distinct from a performance's metadata.
-	 */
-	metadata: ProgramMetadata;
-	/**
-	 * This is the "working" state of the machine, outside of a performance.
-	 * When a user is modifying the program, this is how we store what settings they use.
-	 * Whenever the user saves, this field gets updated, and whether they load, this gets read.
-	 * It has almost no relevance when editing a performance since the performance events will
-	 * cause it to be updated dynamically.
-	 *
-	 * ## Example
-	 * ```typescript
-	 * let prog: Program;
-	 * // invert the muted state of the snare
-	 * prog.state.machine.mute.snare = !prog.state.machine.mute.snare;
-	 * ```
-	 */
-	state: State;
-	/**
-	 * All the [[TickedDropEvent]]s that occur during this program. They *must* be in ascending order by tick.
-	 */
-	dropEvents: TickedDropEvent[];
-}
+//#endregion
 
-/** Represents metadata about a program. Used to indicate how this program should be played. */
-export interface ProgramMetadata {
-	/**
-	 * The title of this program. It can be distinct from the title of the performance.
-	 */
-	title: string;
-	/**
-	 * The author of this program. It can be distinct from the title of the performance.
-	 */
-	author: string;
-	/**
-	 * Pulses per quarter.
-	 * This basically represents how many distinct pieces a quarter note is divisible by.
-	 * The higher the PPQ, the more flexibility when it comes to placing notes.
-	 */
-	readonly tpq: 240;
```

Comment:
Change to "ppq"

Reply:
I think the comment is wrong here: tpq is ticks per quarter